sled-storage/src/manager.rs

```diff
@@ -955,8 +955,8 @@ impl StorageManager {
             err: None,
         };
 
-        let mountpoint_path =
-            config.name.mountpoint(ZPOOL_MOUNTPOINT_ROOT.into());
+        let mountpoint_root = &self.resources.disks().mount_config().root;
```

Comment:
This change is split out as a separate PR here: https://github.com/oxidecomputer/omicron/pull/7110

It's necessary for performing the testing in `sled-agent/src/support-bundle/storage.rs`

Reply:
Merged now; resolving this comment